examples/gatsby/src/components/Layout.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Link, graphql, useStaticQuery } from 'gatsby'
+
+export default function Layout({ children }) {
+  const linkListData = useStaticQuery(graphql`
+    query LinkListsQuery {
+      nacelleSpace {
+        linklists {
+          handle
+          links {
+            title
+            to
+            type
+          }
+        }
+      }
+    }
+`)
+  const mainMenu = linkListData.nacelleSpace.linklists.find(linkList => linkList.handle === 'main-menu')
+  return (
+    <>
+      <nav style={{ display: 'flex', justifyContent: 'space-around', marginBottom: '3em' }}>
```

Comment:
Not a big deal for now, but would be nice in the future to use Emotion for styling here like we are in other examples. I realize it's a bit more overhead, but I think it keeps the styles more separated from the import parts of the example. I could be thinking about that wrong though.

Reply:
Part of me wanted to reach for it, but I didn't because I wanted it to be effectively unstyled, so as to keep focus on the purpose of the example as demonstrating the data layer collection only, and not something that we've put any styling energy into.

I def agree that Emotion is a great way to go, and for future examples that will benefit from more styling (e.g. full Gatsby Image support in v2), I think it would be great.